src/main/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBBase.java

```diff
@@ -41,6 +41,8 @@
 import com.amazonaws.services.dynamodbv2.model.DescribeContinuousBackupsResult;
 import com.amazonaws.services.dynamodbv2.model.DescribeGlobalTableRequest;
 import com.amazonaws.services.dynamodbv2.model.DescribeGlobalTableResult;
+import com.amazonaws.services.dynamodbv2.model.DescribeGlobalTableSettingsRequest;
```

Comment:
global tables are a new feature and thus were added when we upgrade the dynamodb sdk version